### DISCUSSION TITLE
Steam-Client disconnects at 5:00 every day

### DISCUSSION THREAD

**Discussion by tom-jm69** [ValvePython/steam#403] [Q&A] [answered]
Hey,
does someone know why my bot gets in a login loop? it somehow random disconnects and then it tries to log back in and gets disconnected again?

`@sc.on('disconnected')
def disconnect():
    sc.reconnect(2, 30)
    print("Reconnecting...")
    cs.launch()
`
Here are the logs:
[logs.txt](https://github.com/ValvePython/steam/files/9306632/logs.txt)

**Comment by tom-jm69** [comment, marked as answer]
Could you please delete this? Fixed it

**Reply by rossengeorgiev** [reply]
Worth mentioning what the cause was. Even it was a code mistake. It would be useful to others. There is nothing in the library that would cause something like the described issue.
